com.github.simonwhitaker.Gibo.appdata.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
```

Comment:
Should be upstreamed

Reply:
https://github.com/simonwhitaker/gibo/pull/44